bin/check_google_doc_export.rb

```diff
@@ -16,12 +16,30 @@
 require_relative '../lib/article_json'
 
 doc_id = '1E4lncZE2jDkbE34eDyYQmXKA9O26BHUiwguz4S9qyE8'
-url = "https://docs.google.com/feeds/download/documents/export/Export?id=#{doc_id}&exportFormat=html"
+url =
+  "https://docs.google.com/feeds/download/documents/export/Export?id=#{doc_id}&exportFormat=html"
 expected_document = JSON.parse(File.read('spec/fixtures/reference_document_parsed.json'))
 
 exported_doc = Net::HTTP.get(URI.parse(url))
 document = JSON.parse(ArticleJSON::Article.from_google_doc_html(exported_doc).to_json)
 
+# `source_url` (for hosted images) is dynamic, so we need to remove it from the comparison`
+def nullify_source_url(hash)
+  content = hash['content']
+  return unless content.is_a?(Array)
```

Comment:
When we return `nil` here, in the assignment `expected_document = nullify_source_url(expected_document)`, we'll end up with a nil `expected_document`, is this what we want?

Reply:
From printing the value of the `expected_document` I never did return `nil` here, but you are right, I should have been returning the `content`, rather than `nil`.

Anyway, it has been amended to a one-liner that does not include this early return. Thanks.